src/shared/install.c

```diff
@@ -3619,12 +3619,16 @@ static UnitFileList* unit_file_list_free_one(UnitFileList *f) {
         return mfree(f);
 }
 
-Hashmap* unit_file_list_free(Hashmap *h) {
-        return hashmap_free_with_destructor(h, unit_file_list_free_one);
-}
-
 DEFINE_TRIVIAL_CLEANUP_FUNC(UnitFileList*, unit_file_list_free_one);
 
+DEFINE_HASH_OPS_WITH_VALUE_DESTRUCTOR(
+        unit_file_list_hash_ops_free,
+        char,
+        string_hash_func,
+        string_compare_func,
+        UnitFileList,
+        unit_file_list_free_one);
```

Comment:
In the last commit, `unit_file_list_free()` is removed. Then, maybe better to rename `unit_file_list_free_one()` -> `unit_file_list_free()`.